hudi-common/src/main/java/org/apache/hudi/metadata/HoodieTableMetadataUtil.java

```diff
@@ -144,7 +144,7 @@ class ColumnStats {
 
         GenericRecord genericRecord = (GenericRecord) record;
 
-        final Object fieldVal = convertValueForSpecificDataTypes(field.schema(), genericRecord.get(field.name()), true);
+        final Object fieldVal = convertValueForSpecificDataTypes(field.schema(), genericRecord.get(field.name()), false);
```

Comment:
Hi, @alexeykudinkin , can you take a look at this line, the change expects to work correctly right ? (same with parquet files, the parquet max/min for timestamp is the raw long, and we use a `LongWrapper` here).